ksqldb-functional-tests/src/test/resources/sql-tests/query-upgrades/rowpartition-rowoffset.sql

```diff
@@ -0,0 +1,130 @@
+----------------------------------------------------------------------------------------------------
+--@test: upgrade stream across pseudocolumn versions
```

Comment:
I thought we discussed removing this test (and the one below) in https://github.com/confluentinc/ksql/pull/8097#discussion_r717135179?

Reply:
I interpreted this as adding tests, not necessarily removing. I'll go ahead and do that.